fix(ui5-combobox, ui5-multicombobox): center ValueState text

ndeshev opened the pull request:
ValueState text is now vertically centered in the Popover's header.

Issue: #2274

ilhan007 reviewed (changes requested):
Because the ResponsivePopover.css styles take effect in input with suggestions, combobox, multi combobox, multi input, select and all date-time components, I think the value state text alignment of the text should be checked for them as well. 
(1) Check if  this is a bug only in the ComboBox
and 
(2) make sure the fix won't break the rest of the components.

ndeshev commented:
The issue was also present in MultiComboBox, I added a fix for it and reverted back the previous value of the responsive popover's header CSS height param.

ilhan007 commented on packages/main/src/themes/base/sizes-parameters.css:
```diff
@@ -69,9 +69,6 @@
 	--_ui5_textarea_padding: 0.5625rem 0.6875rem;
 
-	/* Responsive Popover */
-	--_ui5-responnsive_popover_header_height: 2.75rem;
```
Should we remove the cozy param? I think we should keep it and just fix the typo - responnsive -> responsive

ilhan007 reviewed (changes requested):
It works for single line, but not for the valueStateMessage slot. I guess it is the line-height

<img width="382" alt="Screenshot 2021-01-27 at 1 15 34 PM" src="https://user-images.githubusercontent.com/15702139/105983684-c2da0000-60a1-11eb-9e5c-178c53c1f9ee.png">
